src/main/java/reactor/core/publisher/TopicProcessor.java

```diff
@@ -256,17 +419,18 @@
 	 * subscribers ?
 	 * @param <E> Type of processed signals
 	 * @return a fresh processor
+	 * @deprecated use {@link Builder#build()}
 	 */
+	@Deprecated
 	public static <E> TopicProcessor<E> create(String name, int bufferSize,
 	                                                WaitStrategy strategy,
 	                                                boolean autoCancel) {
-		return new TopicProcessor<>(name,
-				bufferSize,
-				strategy == null ?
```

Comment:
same as before with strategy (applies basically everywhere...) although a simpler way around that problem would be to apply defaults to non-nullable constructor parameters in `build`, so that eg. calling `.waitStrategy(null)` on the builder still results on a `phasedOffLiteLock` being applied, same for `.name(null)`... Or 3rd option: NPE on null name or strategy.